### ISSUE TITLE
Render images in html

### ISSUE THREAD

**Issue by elnurvl** [irshuLx/Android-WYSIWYG-Editor#18]
In the render mode images are not shown. Is it a bug or the module has not this feature? I am using v2.0.0.

**Comment by glaucioleonardo** [comment]
You should try the node serialization to render the images, but I was not able to upload as the same order.

**Comment by irshuLx** [comment]
@glaucioleonardo , could u share your serialized content?
When u want to insert image, you need to start the image picker intent ( u could also use `editor.openImagePicker(); ` )  to make it easier.

you should then call:   `editor.insertImage(bitmap);`

and add the listener:

```
editor.setEditorListener(new EditorListener() {
            @Override
            public void onTextChanged(EditText editText, Editable text) {
                // Toast.makeText(EditorTestActivity.this, text, Toast.LENGTH_SHORT).show();
            }
            @Override
            public void onUpload(Bitmap image, String uuid) {
                //do your upload operations here. after the upload, obtain the url and pass it along with the uuid.                
editor.onImageUploadComplete("http://www.videogamesblogger.com/wp-content/uploads/2015/08/metal-gear-solid-5-the-phantom-pain-cheats-640x325.jpg",uuid);
               // editor.onImageUploadFailed(uuid);
            }
        });

```

**Comment by irshuLx** [comment]
![github issue screenshot](https://user-images.githubusercontent.com/12872191/27792575-fc2df61e-602c-11e7-802b-39f04b8d1f15.png)

**Comment by irshuLx** [comment]
Sorry, i just realize that it was not rendered on editor. Fixed on 79f476decbefd4cd0c850b09ddae4c7e206cdc3c
